### ISSUE TITLE
The test.py example seems to be out-dated

### ISSUE THREAD

**Issue by dsandber** [cfaessler/python-picaso-lcd#5]
The methods that test.py references don't exist -- probably because you refactored the code into text and touch objects.  If I submit a patch for this will you accept it? [Only want to bother submitting patches if you'll integrate them]

**Comment by dbrgn** [comment]
Actually this code is not even finished. There are many internal functions that are not yet implemented.

It wouldn't be very hard to do so (basically you simply translate the stuff from the datasheet into Python code). The problem is, we had an issue with how the chip works and contacted the company behind it, but never got an answer. So we never continued working on this code.

The main issue was that the serial interface does not seem to be suited for touch input. First you need to define a region for which you want to get events. Then you need to poll the display for touches. That seems absurd. There must be some way to get interrupts (or something similar) on touch events.

This is handled much better when you use their programming language via the SD card. But for our project we need external control. Hardcoding something on the SD card with their windows-only software environment is no solution.

So, until we can resolve this issue I probably won't continue working on this code. A touchscreen LCD without the touch support is not very usable.

**Comment by dbrgn** [comment]
Regarding your request, yes I'll integrate patches but don't really have the time to test them atm. If you provide a clean pull request and confirm that you tested it, I'll merge it :)

**Comment by dsandber** [comment]
Yeah, I'm pretty disappointed with their interface modes as well.  You can choose their bizarre non-standard language which forces you to deal with converting between byte and word pointers, or use the serial interface with all its limitations.  

That said, I'll implement a keyboard (hopefully being able to copy it from their designer tool) and if it works ok with polling than I'll use it.  If they "queue" the touch events on their side it might work alright.

Cool, I'll submit a patch shortly.

**Comment by dsandber** [comment]
Incidentally, you can directly edit their SPE (the code which runs on the LCD and implements the serial interface) yourself.  So you can fairly easily implement an event driven interface if you like.  The only problem is that their custom language is quite frustrating.  Why they decided to write their own language rather than using one that already has millions of users is a great mystery to me :)

**Comment by dbrgn** [comment]
> Incidentally, you can directly edit their SPE (the code which runs on the LCD and implements the serial interface) yourself.

Ah, that's quite nice. So we could in theory create functions that registers events and actions via the serial interface...
